src/config/env/index.js

```diff
@@ -9,11 +9,13 @@ const config = require(`./${env}`);
 const defaults = {
     userRoles: ['guest', 'user:read', 'user:write', 'user', 'admin'],
     root: path.join(__dirname, '/..'),
+    cleanUploads: false,
+    uploadsFolder: './uploads',
```

Comment:
it depends on```cwd```, so if you start application from other folder it would serve files from other directory, which is unexpected behavior